#!/usr/bin/env python
PACKAGE = "racer"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("kp", double_t, 0, "Proportional gain", 0, -1, 1)
gen.add("ki", double_t, 0, "Integral gain", 0, -1, 1)
gen.add("kd", double_t, 0, "Derivative gain", 0, -1, 1)
gen.add("error_tolerance", double_t, 0, "Error tolerance", 0, 0, 1000)

exit(gen.generate(PACKAGE, "racer", "PID"))